waterlog/src/Components/TankLevels.js

```diff
@@ -1,37 +1,36 @@
 import React, { Component } from 'react';
 import Tank from './Tank'
-
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchTankLevels } from './../actions/TankLevelsActions'
 class TankLevels extends Component {
-  constructor(props){
-    super(props);
-    this.state = {
-      tanks: [{
-          name: "Tank 1",
-          waterLevel: 0,
-          status: "sufficient level",
-          pumpOn: true
-        },{
-          name: "Tank 2",
-          waterLevel: 75,
-          status: "insufficient level",
-          pumpOn: false
-        },{
-          name: "Tank 3",
-          waterLevel: 50,
-          status: "acceptable level",
-          pumpOn: true
-        }
-      ]
-    }
-  }
-  render(){
+  componentWillMount() {
+    this.props.fetchTankLevels(this.props.id);
+  };
+  render() {
+    if (this.props.levels.loading)
+      return <p>Loading...</p>
+    if (this.props.levels.error)
+      return <p>Failed to fetch</p>
+
+      console.log(this.props.levels)
     return (
       <div className="tank-container">
-        {this.state.tanks.map((tank,index) => 
-          <Tank key={index} tank={tank} />
+        {this.props.levels.map((tank, index) =>
+          <div onClick={() => window.location = `/usage/tank/${index + 1}`}><Tank key={index} tank={tank} /></div>
```

Comment:
Why not rather have
```js
<Link to="your/url/here">Link Text Goes Here</Link>
```
You can also use the `match.url` field for the base url instead `window.location`.

Reply:
i tryed using <link> but it  wasnt doing want suppose to do .